src/main/java/usi/si/seart/model/MetricLanguage.java

```diff
@@ -31,6 +32,10 @@
 public class MetricLanguage {
 
     @Id
+    @GeneratedValue
+    @Column(name = "language_id")
+    Long language_id;
```

Comment:
Use camel case for entity field names, but snake case for column names. In other words just rename the field to `langaugeId`

Reply:
Renamed to `id`